spec/models/user_spec.rb

```diff
@@ -93,7 +93,7 @@
   end
 
   describe "#volunteers_with_no_supervisor?" do
-    subject { Supervisor.volunteers_with_no_supervisor(casa_org) }
+    subject { User.volunteers_with_no_supervisor(casa_org) }
```

Comment:
I believe that User/Supervisor are equivalent here

Reply:
@compwron 

Thank you. Interestingly, `User` and `Supervisor` behave differently in the tests. Below are some examples. Any suggestions about why that might be? I had assumed it was because `Supervisor` was equivalent to `User.where(type: 'Supervisor')`.

### With `subject { User.volunteers_with_no_supervisor(casa_org) }`
```
$ be rspec /home/cmcintosh/code/rubyforgood/casa/spec/models/user_spec.rb:95
Run options: include {:focus=>true, :locations=>{"./spec/models/user_spec.rb"=>[95]}}
..

Finished in 0.14495 seconds (files took 2.39 seconds to load)
2 examples, 0 failures

Coverage report generated for RSpec to /home/cmcintosh/code/rubyforgood/casa/coverage. 299 / 1488 LOC (20.09%) covered.
```

### With `subject { Supervisor.volunteers_with_no_supervisor(casa_org) }`

```
cmcintosh@clifton-x1-carbon:~/code/rubyforgood/casa ‹issue-449-track-non-active-supervisor-volunteer-relationships›
$ bundle exed rspec /home/cmcintosh/code/rubyforgood/casa/spec/models/user_spec.rb:95
Run options: include {:focus=>true, :locations=>{"./spec/models/user_spec.rb"=>[95]}}
.F

Failures:

  1) User#volunteers_with_no_supervisor? volunteers returns unassigned volunteers
     Failure/Error: expect(subject.map(&:display_name).sort).to eq(["aaa", "bbb", "fff"])
     
       expected: ["aaa", "bbb", "fff"]
            got: []
     
       (compared using ==)
     # ./spec/models/user_spec.rb:115:in `block (4 levels) in <top (required)>'

Finished in 0.1535 seconds (files took 2.39 seconds to load)
2 examples, 1 failure

Failed examples:

rspec ./spec/models/user_spec.rb:114 # User#volunteers_with_no_supervisor? volunteers returns unassigned volunteers
```